public/data/vinay.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Vinay",
+    "type": "personal",
+    "bio": "I am a web developer and an open-source enthusiast.",
+    "avatar": "https://github.com/Vinay0910.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Vinay0910",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Vinayshelke15",
+        "icon": "twitter"
+      }
+    ]
+  }
+  
```

Comment:
Looks like there's an extra line here. GitHub doesn't display the endline; thus, this line should be removed.
```suggestion
```

Reply:
I guess it's resolved nkw